lib/Conversion/FIRRTLToHW/LowerToHW.cpp

```diff
@@ -265,6 +266,19 @@ struct CircuitLoweringState {
     binds.push_back(op);
   }
 
+  /// For a given Type Alias, return the corresponding AliasType. Create and
+  /// record the AliasType, if it doesn't exist.
+  hw::TypeAliasType getTypeAlias(Type rawType, BaseTypeAliasType firAliasType,
```

Comment:
nit: Can we make argument order consistent with addTypedecl?

Reply:
Thanks for spotting this, fixed.